classes/order/Order.php

```diff
@@ -2681,6 +2681,30 @@ public function getOrderDetailTaxes()
         );
     }
 
+    /**
+     * @param int $productId
+     * @param int $productAttributeId
+     *
+     * @return int|null
+     */
+    public function getProductSpecificPriceId(int $productId, int $productAttributeId): ?int
```

Comment:
I added this method because the `AbstractOrderHandler` and `OrderFeatureContext` didn't fetch the SpecificPrice ID the exact same way and it created errors and false positive/negative in the behat suite

So now they both use this method which ensures they both use the correct values, if this needs to be modified we won't have to update both classes now